<launch>

	<node type="rviz" name="rviz_station_keeping" pkg="rviz" args="-d /home/.rviz/station_keeping.rviz" />

	<node pkg="rostopic" type="rostopic" name="gps_origin" args="pub /launch_send_gps_origin geometry_msgs/Vector3 -- 50.695396 -4.236313 0" />
	<node name="zone_to_stay" pkg="plymouth2020" type="zone_to_stay.py" args="--test" respawn="true" output='screen' />
	<node name="simu_boat" pkg="plymouth2020" type="simu_boat.py" args="--test" respawn="true" output='screen' />
	<node name="controler_station_keeping" pkg="plymouth2020" type="controler_station_keeping.py" args="--test" respawn="true" output='screen' />
	<node name="rviz_displayer_station_keeping" pkg="plymouth2020" type="rviz_displayer_station_keeping.py" args="--test" respawn="true" output='screen' />

</launch>